hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1258,6 +1258,15 @@
       // Timeout based on a ~7 minute dvsim runtime.
       run_opts: ["+sw_test_timeout_ns=7_000_000"]
     }
+    {
+      name: chip_sw_rv_core_ibex_lockstep_glitch
+      uvm_test_seq: chip_sw_rv_core_ibex_lockstep_glitch_vseq
```

Comment:
Nit: should we start calling it `top_`

Reply:
In general I agree test names should be changed to realign with the "*top*-level" nomenclature. However, I think this PR, which adds a new test, should not break with the naming pattern of all other test names, but rather a separate PR should change the names of all tests (and deal with the potential consequences) -- but that is beyond the scope of this PR (and potentially even beyond M2) IMO.

Reply:
Yes, that makes sense. We should leave it as `chip_` for now.